UmbracoFood.Core/Interfaces/IOrderRepository.cs

```diff
@@ -16,10 +16,12 @@ public interface IOrderRepository
 
         IEnumerable<Status> GetStatuses();
 
-        void AddOrderMeal(OrderedMeal orderedMeal);
+        void AddOrderedMeal(OrderedMeal orderedMeal);
 
         void ChangeStatus(int orderId, OrderStatus newStatus);
 
         void SetOrderIsInDelivery(int orderId, DateTime estimatedDeliveryTime);
+
+        void RemoveOrderedMeal(int orderedMealId);

```

Comment:
jak będzie autoryzacja domenowa to wtedy owner lub osoba dodająca posiłek będzie mogła go usunąć. A takie podawanie orderId to żadne zabezpieczenie.